src/smart-components/Channel/components/SuggestedMentionList/index.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState, useEffect, useContext } from 'react';
+import SendBird from 'sendbird';
+import './index.scss';
+
+import Label, { LabelTypography, LabelColors } from '../../../../ui/Label';
+import Icon, { IconTypes, IconColors } from '../../../../ui/Icon';
+import SuggestedUserMentionItem from './SuggestedUserMentionItem';
+import { useChannel } from '../../context/ChannelProvider';
+import useSendbirdStateContext from '../../../../hooks/useSendbirdStateContext';
+import { LocalizationContext } from '../../../../lib/LocalizationContext';
+import { MAX_USER_MENTION_COUNT, MAX_USER_SUGGESTION_COUNT, USER_MENTION_TEMP_CHAR } from '../../context/const';
+import { MessageInputKeys } from '../../../../ui/MessageInput/const';
+
+export interface SuggestedMentionListProps {
+  targetNickname: string;
+  memberListQuery?: Record<string, string>;
+  onUserItemClick?: (member: SendBird.User) => void;
+  onFocusItemChange?: (member: SendBird.User) => void;
+  onFetchUsers?: (users: Array<SendBird.User>) => void;
+  renderUserMentionItem?: (props: { user: SendBird.User }) => JSX.Element;
+  ableAddMention: boolean;
+  maxMentionCount?: number;
+  maxSuggestionCount?: number;
+  inputEvent?: React.KeyboardEvent<HTMLDivElement>;
+}
+
+const DEBOUNCING_TIME = 300;
```

Comment:
nit: should add this as a prop - can do next release

Reply:
Yes, that's in our plan